content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -292,68 +307,138 @@ generator client {
 
-Prisma Migrate is a CLI for declarative data modeling & migrations. Unlike most migration tools that come as part of an ORM, you only need to describe the current schema, instead of the operations to move from one state to another. Prisma Migrate infers the operations and carries out the migration for you.
+Prisma Migrate is a CLI for declarative data modeling & migrations. Unlike most migration tools that come as part of an ORM, you only need to describe the current schema, instead of the operations to move from one state to another. Prisma Migrate infers the operations, generates the SQL and carries out the migration for you.
 
-This example demonstrates using Prisma in a new project with a new database schema similar to the Sequelize example above:
+This example demonstrates using Prisma in a new project with a new database schema similar to the blog example above:
 
 1. Create the Prisma schema:
 
 ```prisma
 // schema.prisma
 datasource db {
-  provider = "sqlite"
-  url      = "file:data.db"
+  provider = "postgres"
+  url      = "postgresql://janedoe:janedoe@localhost:5432/hello-prisma"
 }
 
 generator client {
   provider = "prisma-client-js"
 }
 
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  content   String? @map("post_content")
+  published Boolean @default(false)
+  author    User?   @relation(fields: [authorId], references: [id])
+  authorId  Int?
+}
+
 model User {
-  id        Int      @id
-  birthDate DateTime
-  email     String   @unique
-  firstName String?
-  lastName  String?
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
 }
 ```
 
-2. `prisma migrate save --experimental`: Save the migration. Typically migrations are saved in the code repository
-3. `prisma migrate up --experimental`: Run the migration which will create the database schema
-4. `prisma generate`: Re-generate Prisma Client
+2. Run `prisma migrate` to generates the SQL for the migration, apply it to the database, and generate Prisma Client.
 
 For any further changes to the database schema:
 
-1. Apply changes to the Prisma schema, e.g. add a `registrationDate` field to the `User` model
-1. Follow steps 2-4.
+1. Apply changes to the Prisma schema, e.g., add a `registrationDate` field to the `User` model
+1. Run `prisma migrate` again.
 
-The last two steps demonstrate how declarative migrations work by adding a field to the Prisma schema and using Prisma Migrate to transform the database schema to the desired state. Lastly, the Prisma Client was regenerated so that it reflects the updated schema.
+The last step demonstrates how declarative migrations work by adding a field to the Prisma schema and using Prisma Migrate to transform the database schema to the desired state. After the migration is run, Prisma Client is automatically regenerated so that it reflects the updated schema.
```

Comment:
Is it true already that Migrate generates the new client?

Reply:
The early access one generates the client automatically. 

I've updated it to: 
> After the migration is run, you generate Prisma Client (with `prisma generate`) so that it reflects the updated schema.